examples/apps/ai-collab/src/components/TaskGroup.tsx

```diff
@@ -52,14 +54,116 @@ export function TaskGroup(props: {
 	const [isAiTaskRunning, setIsAiTaskRunning] = useState<boolean>(false);
 	const [llmBranchData, setLlmBranchData] = useState<{
 		differences: Difference[];
-		originalBranch: BranchableTree;
-		forkBranch: TreeBranchFork;
-		forkView: TreeView<typeof SharedTreeAppState>;
+		originalBranch: TreeViewAlpha<typeof SharedTreeAppState>;
+		forkBranch: TreeViewAlpha<typeof SharedTreeAppState>;
```

Comment:
Nit: I had named these `<something>Branch` vs the removed `forkView` to make the distinction clear between the two kinds of objects. If the ST API changes let us now work directly with the views, we can probably update the property names as well.

Reply:
Slight adjustment to naming in commit a414f8c75c84070e4f42704b3f61856b2633f2d2 , not sure if this captures your intention but I think it'll be easier for users less familiar with branching.